### ISSUE TITLE
Clustering errors

### ISSUE THREAD

**Issue by chrissoria** [chrissoria/BICS_Political_Polarization#4]
Does it make sense to cluster errors in any way?

Possible levels to cluster:

- County
- State
- Wave

PS across all pooled models, wave is significantly associated with measures of interest (stated contact reduction, concern, reported contacts, masked contacts)

**Comment by dfeehan** [comment]
Yes, I think we will want to cluster in at least some of those ways. But, as with #3, I think it will make sense to focus on the substantive question a little more before we go into the modeling details.
